test/functional/Java21andUp/src/org/openj9/test/jep425/VirtualThreadTests.java

```diff
@@ -0,0 +1,313 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2023
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package org.openj9.test.jep425;
+
+import org.testng.annotations.Test;
+import org.testng.Assert;
+import org.testng.AssertJUnit;
+import static org.testng.Assert.fail;
+
+import java.lang.reflect.*;
+import java.lang.Thread;
+import java.time.Duration;
+import java.util.concurrent.Executor;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.locks.LockSupport;
+import java.util.stream.IntStream;
+
+/**
+ * Test cases for JEP 425: Virtual Threads (Preview) Continuation execution
```

Comment:
The changes in this PR doesn't reflect the latest FFI API changes as specified in https://github.com/eclipse-openj9/openj9/issues/16951 which have not yet merged to the baseline in OpenJDK. So we have to disable all Java20 test suites in advance once the new changes in Java21 come up.

Reply:
We need to find a way to avoid creating multiple copies of tests: that practice is not scalable. I created https://github.com/eclipse-openj9/openj9/issues/17206 to track that goal.

Reply:
FYI: @fengxue-IS